packages/gravity-ui-nunjucks/gulp/patternlab.js

```diff
@@ -0,0 +1,216 @@
+/******************************************************
+ * EXPORTS PATTERN LAB TASKS
+ *
+ * Note that substantial chunks of this file were
+ * originally copied from Pattern Lab's Node Gulp
+ * Edition gulpfile:
+ * https://github.com/pattern-lab/edition-node-gulp/blob/master/gulpfile.js
+ *
+******************************************************/
+
+const gulp = require('gulp');
+const sass = require('gulp-sass');
+const eyeglass = require('eyeglass');
+const argv = require('minimist')(process.argv.slice(2));
+
+const pkgPaths = require('./paths.js');
+const taskNamePrefix = 'pattern-library:';
+
+//read all paths from our namespaced config file
+const config = require('../patternlab-config.json');
+const patternlab = require('@pattern-lab/core')(config);
+
+const plPaths = config.paths;
+
+// Exported array of glob patterns that clean tasks can use to
+// delete any files generated at build-time
+const generatedFileGlobs = [
+  pkgPaths.pkgRootPath('dependencyGraph.json'),
+  pkgPaths.pkgRootPath(plPaths.source.meta, '*.mustache')
+];
+
+
+
+
+/******************************************************
+ * STYLEGUIDE CSS TASKS
+******************************************************/
+
+function buildSass() {
+  return gulp.src(pkgPaths.pkgRootPath(plPaths.source.root, 'sass', 'pattern-scaffolding.scss'))
+    .pipe(sass(eyeglass(sass.sync().on('error', sass.logError))))
+    .pipe(gulp.dest(pkgPaths.pkgRootPath(plPaths.public.css)))
+};
+buildSass.displayName = taskNamePrefix + 'sass:build';
```

Comment:
Should probably use string interpolation $`My {var}` instead of `+` combining throughout the file.